source/usage-examples/find.txt

```diff
@@ -0,0 +1,61 @@
+=======================
+Find Multiple Documents
+=======================
+
+.. default-domain:: mongodb
+
+You can query for multiple documents in a collection by calling the ``find()``
+method on a ``MongoCollection`` object. You can pass a query filter to the
```

Comment:
```suggestion
method on a ``MongoCollection`` object. Pass a query(TODO: link to filter builder) filter to the
```
I think that removing `You can` reduces some redundancy here, and is directive. Thoughts?

Reply:
Fine with this. I think I had commented about not using "You can" twice in succession on a NodeJS PR (there are multiple instances of it in the NodeJS Usage Examples) but that was rejected. It might be worthwhile revisiting in the future.